src/substitution.js

```diff
@@ -12,7 +12,10 @@ import Layout from './layout';
  * @constructor
  */
 function Substitution(font) {
-    Layout.call(this, font, 'gsub');
+    Layout.call(this, font, 'gsub', [
+        { featureName: 'rlig',  supportedLookups: [4] }
+        // TODO: Define all supported features to use layout.getFeaturesLookups for a sequence ordered feature lookups
```

Comment:
Is this a TODO for some separate future PR, or did you forget to implement this? How important is this, shouldn't it be in this PR?

Reply:
@Connum this configuration param is used by the `Layout.getFeaturesLookups` that implements the ordered features lookup sequence. The thing is I have improved only Positioning features lookup (`Position.prototype.getPositionFeatures`) leaving substitutions untouched. So for the substitutions still uses the  "old features lookups parsing flow" which is as it was before. Adapting this ordered lookup sequence to the substitution will require a decent refactoring (and then this line will be "in use"). 

Hope this make sense?

Reply:
PS. and just to be on the same page, this array defines which lookup tables are supported for which features. This way we can have less redundancy and a single algorithm that parses the lookup tables. That why `Layout.getFeaturesLookups` got implemented.

We might get refactor **substitutions** with better tests coverage. Right now I have not taken this call because I had some concerns about the tests coverage not mature enough for such change to the substitutions (bidi)